src/test/java/com/klodnicki/Bike/v2/service/ChargingStationServiceHandlerTest.java

```diff
@@ -96,13 +98,12 @@ public void findById_ShouldReturnStationForAdminResponseDTO_WhenExistInDatabase(
     @Test
     public void findStationById_ShouldReturnChargingStation_WhenExistInDatabase() {
         //given
-        chargingStationRepository.deleteAll();
         ChargingStation chargingStation = new ChargingStation();
+        chargingStation.setBikeList(new ArrayList<>());
         chargingStationRepository.save(chargingStation);
 
         //when
         ChargingStation actual = chargingStationServiceHandler.findStationById(chargingStation.getId());
-        actual.setBikeList(null);
 
         //then
         assertEquals(chargingStation, actual);
```

Comment:
My suggestion for this method:
```
//given
ChargingStation chargingStation = chargingStationRepository.save(new ChargingStation());

//when
ChargingStation actual = chargingStationServiceHandler.findStationById(chargingStation.getId());

//then
assertEquals(chargingStation, actual);
```

Reply:
Ok I already corrected equals() and hashcode() and everything works fine